src/pages/projectSettings/tabs/EventLogTab/EventLogDiffs.test.ts

```diff
@@ -0,0 +1,203 @@
+import { getEventDiffLines } from "./EventLogDiffs";
+
+const before = {
```

Comment:
Can we trim these objects down to the relevant fields for the test? That will help clarify what we're testing I think

Reply:
done